src/passing-test-cases.scss

```diff
@@ -107,6 +107,19 @@ $my-var: 10px;
   color: #f00;
 }
 
+// Nesting Depth test
+$colors: {
```

Comment:
Let's switch this to the following:

```
$colors: (
  white: #fff
);
```

The current code introduces lint in the passing-test-cases.scss file.

Reply:
Thanks, not sure how I missed this ^^'